app/components/admin/appropriate_bodies/bulk_upload_component.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= batch_cards %>
+
+<%= batch_details %>
+
+<%=
+  govuk_tabs do |tabs|
+    tabs.with_tab(label: "Errors") do %>
+      <% if error_details.render? %>
+        <%= error_details %>
+      <% else %>
+        <h3 class='govuk-heading-m'>No errors</h3>
+      <% end %>
+    <% end if error_details?
+
+    tabs.with_tab(label: "Submitted inductions") do %>
+      <% if induction_details.render? %>
+        <%= induction_details %>
+      <% else %>
+        <h3 class='govuk-heading-m'>No inductions</h3>
+      <% end %>
```

Comment:
It feels a little unconventional to be calling the `render?` method like this.

I wonder whether it's worth moving the "No inductions" branch inside the component 🤔

What do you think?